### PR TITLE
bht: cleanup

### PR DESCRIPTION
**Pull request by FFHener**
As we will swap out the Router and Switch anyway I thought I would bring the location closer to our convention. Should we add some Nearfieldstuff as well? Does it make sense there?

This PR will be deployed On-Site at 7th of November

EDIT: Changed mgmt-IP according to https://github.com/freifunk-berlin/bbb-configs/issues/983

### CODE REVIEW

File: locations/bht.yml [hunk 1]
```diff
@@ -122,94 +124,98 @@ airos_dfs_reset:
...
+    prefix: 10.230.23.137/32
+    ipv6_subprefix: -19
 
-  - vid: 104
+  - vid: 40
     role: dhcp
-    prefix: 10.230.23.152/29
+    prefix: 10.230.23.144/28
     ipv6_subprefix: 0
     untagged: true
```
Comment (Noki) [inline locations/bht.yml:206]: let's untag a switchport instead

**Comment by pmelange** [comment]
There is no need for near field at this location. And therefore we don't need a bigger dhcp network. The old 4909 was hardly ever used because of it's location. It was good to have when doing maintenance. Now with the rb750gr3 we have to make sure that people bring a network cable with them.

**Comment by pmelange** [comment]
And why do we want to use 42 for the mgmt vlan? Wasn't that a problem that everywhere 42 is being used?

**Comment by pmelange** [comment]
I like the rb759gr3. Keep it.

But why remove jup from the list? One of the goals on Thursday is to get jup dish back online.

**Comment by Noki** [comment]
The connection to jup should stay for sure and we need to bring that one up again. In addition we should change mgmt to something in the 4xx range to prepare for 60 GHz so we don't need to touch it again if we build 60 GHz later on. Please use the first free one in the list: https://github.com/freifunk-berlin/bbb-configs/issues/983

File: locations/bht.yml (hunk 1 above)
Comment (Noki) [reply locations/bht.yml:206]: I think the only reason was that switch ports were full and the core had free ports where you could plug in and therefore it was untagged.

**Review by Noki — APPROVED** [review]
looks good to me

**Pull request merged**